packages/react/presence/src/Presence.tsx

```diff
@@ -96,33 +101,20 @@ function usePresence(present: boolean) {
           send('ANIMATION_END');
         }
       };
-
-      const handleAnimationStart = (event: AnimationEvent) => {
-        if (event.target === node) {
-          // if animation occurred, store its name as the previous animation.
-          prevAnimationNameRef.current = getAnimationName(stylesRef.current);
-        }
-      };
-
-      node.addEventListener('animationstart', handleAnimationStart);
```

Comment:
I remembered that we added all this stuff before for a reason so I went through GitHub history and validated the issues that this originally solved are still working here.